EIPS/eip-3074.md

```diff
@@ -177,7 +177,7 @@ TODO
 
 ## Implementation
 
-TODO
+https://github.com/quilt/go-ethereum/tree/eip-3074
 
```

Comment:
`Implementation` section is no longer an EIP section.  Also, external links to third party repositories should not be used.  Especially transient branches.

Remember: EIPs are technical specifications and not an appropriate place for coordination.